core/processor/batch_market_instructions_processor.go

```diff
@@ -124,10 +131,28 @@ func (p *BMIProcessor) ProcessBatch(
 	// these need to be determinitistic
 	idgen := idgeneration.New(determinitisticID)
 
+	failedMarkets := map[string]error{}
+	for _, umm := range batch.UpdateMarginMode {
```

Comment:
Should we add some sort of validation to ensure that at most 1 update margin mode is submitted, per market?

Reply:
idk, is there a requirement for that?

Reply:
I don't know, but it would make sense to me.